packages/esm-active-visits-app/src/visits-summary/visit-detail.test.tsx

```diff
@@ -96,27 +101,22 @@ describe('VisitDetailComponent', () => {
             orders: [],
           },
         ],
+        startDatetime: '2023-07-30T12:34:56Z',
+        visitType: { display: 'Some Visit Type', uuid: 'some-visit-type-uuid' },
+        uuid: defaultProps.visitUuid,
       },
+      error: undefined,
       isLoading: false,
+      isValidating: false,
     });
 
-    render(<VisitDetailComponent visitUuid={visitUuid} patientUuid={patientUuid} />);
-
-    act(() => {
-      screen.getByText('Visit Summary').click();
-    });
+    renderVisitDetail();
 
+    await user.click(screen.getByText('Visit Summary'));
     expect(screen.getByRole('tablist', { name: 'Visit summary tabs' })).toBeInTheDocument();
   });
-
-  it('should render loading indicator when data is loading', () => {
-    mockedUseVisit.mockReturnValue({
-      visit: null,
-      isLoading: false,
-    });
-
-    render(<VisitDetailComponent visitUuid={visitUuid} patientUuid={patientUuid} />);
-
-    expect(screen.queryByRole('button', { name: 'All Encounters' })).not.toBeInTheDocument();
-  });
 });
+
+function renderVisitDetail() {
+  render(<VisitDetailComponent {...defaultProps} />);
+}
```

Comment:
I'm not a big fan of these kinds of functions. A one-liner like that probably doesn't need to be factored out at all. These functions are useful when the thing to be rendered involves nesting components or context or other more complex stuff.